src/app/chat/views/chat_item.nim

```diff
@@ -171,6 +171,13 @@ QtObject:
   proc muted*(self: ChatItemView): bool {.slot.} =
     return ?.self.chatItem.muted
 
+  proc joined*(self: ChatItemView): string {.slot.} = 
+    if self.chatItem.isNil: return ""
+    return $self.chatItem.joined
+
+  QtProperty[string] joined:
+    read = joined
+
```

Comment:
What do you think about passing this to QML as `int` so that no parsing on QML side is needed?

Reply:
`joined` is int64
we may have this converted and exposed as seconds, but this would affect the precision

Reply:
Also looks like we do this already with `nextMsgTimestamp` and `prevMsgTimestamp`